recipe-client-addon/test/browser/browser_PreferenceExperiments.js

```diff
@@ -752,8 +805,8 @@ decorate_task(
     mockPreferences.set("fake.preference", "changed value");
     await PreferenceExperiments.init();
     ok(stopStub.calledWith("test"), "Experiment is stopped because value changed");
-    is(Preferences.get("fake.preference"), "changed value", "Preference value was not changed");
-  }
+    ok(Preferences.get("fake.preference"), "changed value", "Preference value was not changed");
```

Comment:
Err, this is a strange change. `ok` only takes 1 test argument, and its value should be truthy for it to pass. Is this just a mistake? :-)